js/main.js

```diff
@@ -0,0 +1,53 @@
+const NAMES_ARRAY = [
```

Comment:
Нет необходимости указывать тип данных внутри переменной s на конце и ide уже нам расскажет, что это скорее всего массив или какая-то коллекция:
```suggestion
const NAMES = [